landscape.yml

```diff
@@ -1,485 +1,420 @@
+        logo: infolysis.svg
+      - item:
+        name: Infosys
+        homepage_url: https://www.infosys.com/
+        logo: infosys.svg
+      - item:
+        name: Intel
+        homepage_url: https://www.intel.com/
+        logo: intel.svg
+      - item:
+        name: IPification
+        homepage_url: https://www.ipification.com/
+        logo: ipification.svg
+      - item:
+        name: Iquall Networks
+        homepage_url: https://www.iquall.net/
+        logo: iquall.svg
+      - item:
+        name: KDDI
+        homepage_url: https://www.kddi.com/
+        logo: kddi.svg
+      - item:
+        name: Lenovo
+        homepage_url: https://www.lenovo.com/
+        logo: lenovo.svg
+      - item:
+        name: LotusFlare
+        homepage_url: https://www.lotusflare.com/
+        logo: lotusflare.svg
+      - item:
+        name: Matrix Software
+        homepage_url: https://www.matrixx.com/
+        logo: matrixx.svg
+      - item:
+        name: Mavenir
+        homepage_url: https://www.mavenir.com/
+        logo: mavenir.svg
+      - item:
+        name: Netcracker
+        homepage_url: https://www.netcracker.com/
+        logo: netcracker.svg
+      - item:
+        name: NGMN Alliance
+        homepage_url: https://www.ngmn.org/
+        logo: ngmn.svg
+      - item:
+        name: NOS
+        homepage_url: https://www.nos.pt/
+        logo: nos.svg
+      - item:
+        name: NTT DATA
+        homepage_url: https://www.nttdata.com/
+        logo: nttdata.svg
+      - item:
+        name: Open Sesame
+        homepage_url: https://www.opensesame.com/
+        logo: opensesame.svg
+      - item:
+        name: Oracle
+        homepage_url: https://www.oracle.com/
+        logo: oracle.svg
+      - item:
+        name: Quobis
+        homepage_url: https://www.quobis.com/
+        logo: quobis.svg
+      - item:
+        name: Radisys
+        homepage_url: https://www.radisys.com/
+        logo: radisys.svg
+      - item:
+        name: Reply
+        homepage_url: https://www.reply.com/
+        logo: replylive.svg
+      - item:
+        name: Sekura
+        homepage_url: https://www.sekura.com/
+        logo: sekura.svg
+      - item:
+        name: Simptel
+        homepage_url: https://www.siptel.com.sg/
+        logo: simptel.svg
+      - item:
+        name: Singtel
+        homepage_url: https://www.singtel.com/
+        logo: singtel.svg
+      - item:
+        name: SK Telecom
+        homepage_url: https://www.sktelecom.com/
+        logo: sktelecom.svg
+      - item:
+        name: Spry Fox Networks
+        homepage_url: https://www.spryfoxnetworks.com/
+        logo: spryfox.svg
+      - item:
+        name: StarHub
+        homepage_url: https://www.starhub.com/
+        logo: starhub.svg
+      - item:
+        name: Saudi Telecom Company (STC)
+        homepage_url: https://www.stc.com.sa/
+        logo: stc.svg
+      - item:
+        name: Summit
+        homepage_url: https://camaraproject.org
```

Comment:
I wasn't able to find the correct homepage url for this company. Could you please take a look at it? 🙂